src/index.ts

```diff
@@ -49,6 +57,40 @@ const getArgOptions = (argv: string[]): string[] => {
   return options;
 };
 
+const isNodeProject = (rootDir: string): boolean => fs.existsSync(path.join(rootDir, 'package.json'));
+
+const confirmRootDir = async (rootDir: string): Promise<string> => {
+  console.error(`${colors.yellow('Warning:')} Current working directory is not a node project and contains some files.`);
+
+  const answers = await prompt([
+    {
+      type: 'list',
+      name: 'confirm',
+      message: 'Do you wish to initialize your Nightwatch project here?',
+      choices: [
+        {name: 'Yes', value: true},
+        {name: 'No', value: false}
+      ],
+      default: false
+    },
+    {
+      type: 'input',
+      name: 'newRoot',
+      message: 'Relative/absolute path to your existing/new project\'s root directory?',
+      default: '.',
+      when: (answers) => !answers.confirm
+    }
+  ]);
+  // Insert a blank line after prompt.
+  console.error();
```

Comment:
Any reason for using `error` for inserting blank line?

Reply:
Because `error` is being used everywhere to output in the terminal, I also used it here to insert a blank line.